app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingActivityView.java

```diff
@@ -0,0 +1,31 @@
+package com.example.parkingsystemjava.mvp.view;
+
+import android.app.Activity;
+import android.app.FragmentManager;
+import android.content.Context;
+import androidx.annotation.Nullable;
+import java.lang.ref.WeakReference;
+
+public class ParkingActivityView {
```

Comment:
I dont think this name could be renamed...
this view its going to be the parent  for all your future  activity views. So i think it should be called BaseActivityView, or ActivityView. 

its not to bad, but i think you might confuse with ParkingView

Reply:
done, changed the name from "ParkingActivityView" to "ActivityView".